.github/workflows/dotnetcore.yml

```diff
@@ -20,4 +20,4 @@ jobs:
     - name: Build with dotnet
       run: dotnet build --configuration Release
     - name: Test with dotnet
-      run: dotnet test ./Clean.Architecture.sln --configuration Release
+      run: dotnet test ./Clean.Architecture.1.sln --configuration Release
```

Comment:
What are the `.1` additions for?

Reply:
.sln file name should match with `"sourceName"` from `template.json`. `.1` is one option to respect special symbol (like `-`) to correctly instantiate solution from template. Another option is to use `_Clean.Architecture_` everywhere